sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -140,9 +142,12 @@ We’ve made a few other improvements too:
 - The `error-stack` documentation has been greatly improved and we’re always open to more feedback ([create an Issue](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack) or [join us in Discord](https://discord.com/invite/BPMrGAhjPh)).
 - `Termination` has been implemented for `Report`s: it will return the `ExitCode` provided by any `Context` if present, or `ExitCode::FAILURE` will be returned.
 - Compatibility for `anyhow` and `eyre` has been added to convert their types into `Report`.
+
   - We’ve provided a trait to convert `Result<_, anyhow::Error>` or `Result<_, eyre::Report>` to `Result<_, Report<anyhow::Error>>` or `Result<_, Report<eyre::Report>>`.
   - The additional sources of `anyhow::Error` and `eyre::Report` are also attached as frames to maintain their history.
 
+  This will help testing out `error-stack` in existing code bases and will also simplify migrating to `error-stack` is desired.
```

Comment:
```suggestion
  This will help testing out `error-stack` in existing code bases and will also simplify migrating to `error-stack`.
```

Reply:
See this comment for alternative: https://github.com/hashintel/hash/pull/1146#discussion_r985667589

Reply:
Applied in 38d1f9007ef83eb2cb28cd82b16b9e8e531e87f8

Reply:
I would have kept it in the new phrasing, but don't feel strongly.

Reply:
I think this will simplify migrating in any case, even if users don't want to use it 🙂